spec/requests/api/service_templates_spec.rb

```diff
@@ -13,7 +13,7 @@
   let(:ra1)        { FactoryGirl.create(:resource_action, :action => "Provision", :dialog => dialog1) }
   let(:ra2)        { FactoryGirl.create(:resource_action, :action => "Retirement", :dialog => dialog2) }
 
-  let(:picture)    { FactoryGirl.create(:picture, :extension => "jpg") }
+  let(:picture)    { FactoryGirl.create(:picture, :extension => "jpg", :content => 'foo') }
```

Comment:
I'm assuming that this needed to be added because it fails to create now that `:content` is required. I think it would be a good idea to add this to the factory, and then this doesn't need to be changed (assuming that it doesn't test any behavior that's specific to the content)